features/networking/service.feature

```diff
@@ -154,3 +154,43 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-11645
+  Scenario: Create loadbalancer service
+    Given I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/e3e88a1b3706a7ede5c6cb7805fa72d3218a9998/networking/ping_for_pod_containerPort.json |
+    Then the step should succeed
+
+    # Create loadbalancer service
+    When I run the :create_service_loadbalancer client command with:
+      | name | hello-pod |
+      | tcp  | 5678:8080       |
+    Then the step should succeed
+
+    # Get the external ip of the loadbaclancer service
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
+      | resource | svc |
+      | resource_name | hello-pod |
+      | template | {{(index .status.loadBalancer.ingress 0).hostname}} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :service_hostname clipboard
+    """
+
+    # check the external:ip of loadbalancer can be accessed
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given 1 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(0).name` is stored in the :pod1 clipboard
```

Comment:
We can remove this line, and simply use "When I execute on the pod:" below.

Reply:
fix it, thanks!